### PR TITLE
minor fix on readme

### PR DESCRIPTION
**Pull request by dshukertjr**
## What kind of change does this PR introduce?

Cleanup and fix up of examples on Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ class MyWidget extends StatefulWidget {
 
 class _MyWidgetState extends State<MyWidget> {
   // Persisting the future as local variable to prevent refetching upon rebuilds. 
-  final Future<PostgrestResponse<dynamic>> _future = supabase
+  final Future<List<Map<String, dynamic>>> _future = supabase
```
Comment (dshukertjr) [inline README.md:87]: Was using the wrong return type here.

File: README.md [hunk 2]
```diff
@@ -120,11 +120,13 @@ class _MyWidgetState extends State<MyWidget> {
...
-      schema: 'public',
-      table: 'countries'
-    ), (payload, [ref]) {
+    supabase.channel('my_channel').on(
+        RealtimeListenTypes.postgresChanges,
+        ChannelFilter(
+            event: '*',
+            schema: 'public',
+            table: 'countries'),
+        (payload, [ref]) {
```
Comment (dshukertjr) [inline README.md:129]: No code change, just better looking indentation.

File: README.md [hunk 3]
```diff
@@ -163,11 +165,11 @@ class MyWidget extends StatefulWidget {
 
 class _MyWidgetState extends State<MyWidget> {
   // Persisting the future as local variable to prevent refetching upon rebuilds.
-  final _stream = supabase.from('countries').stream(primaryKey: ['id']);
+  final List<Map<String, dynamic>> _stream = supabase.from('countries').stream(primaryKey: ['id']);
 
   @override
   Widget build(BuildContext context) {
-    return StreamBuilder(
+    return StreamBuilder<List<Map<String, dynamic>>>(
```
Comment (dshukertjr) [inline README.md:172]: Just adding return types here.

**Review by Vinzent03 — APPROVED** [review]
LGTM!

**Pull request merged**